### PR TITLE
Remove 'unique' from any CRDs

### PR DESCRIPTION
**Pull request by MarcelMue**
## Checklist

- [ ] Consider SIG UX feedback.
- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by njuettner — APPROVED** [review]
lgtm

File: hack/patches.go [hunk 1]
```diff
@@ -11,15 +11,15 @@ import (
 func patchCAPIWebhook(crd *v1.CustomResourceDefinition) {
 	port := int32(9443)
 	if _, ok := crd.Annotations["cert-manager.io/inject-ca-from"]; ok {
-		crd.Annotations["cert-manager.io/inject-ca-from"] = "giantswarm/cluster-api-core-unique-webhook"
+		crd.Annotations["cert-manager.io/inject-ca-from"] = "giantswarm/cluster-api-core-webhook"
```
Comment (tuommaki) [inline hack/patches.go:14]: Not sure if AWS is running `master`, but this doesn't match still: https://github.com/giantswarm/cluster-api-core-app/blob/main/helm/cluster-api-core/templates/webhook-certificate.yaml#L6

Comment (njuettner) [reply hack/patches.go:14]:
``` 
kg describe cert cluster-api-core-webhook
Name:         cluster-api-core-webhook
Namespace:    giantswarm
```

Comment (njuettner) [reply hack/patches.go:14]: looks good I think

Comment (tuommaki) [reply hack/patches.go:14]: Scratch that, I'm wrong. Didn't notice the separate webhook resource name. This should be fine.

**Review by tuommaki — APPROVED** [review]

**Comment by MarcelMue** [comment]
The linter errors seem unrelated. Will ignore for now as this is urgent.

**Pull request merged**